tests/lib/prepare.sh

```diff
@@ -656,13 +705,15 @@ EOF
     # mount fresh image and add all our SPREAD_PROJECT data
     kpartx -avs "$IMAGE_HOME/$IMAGE"
     # FIXME: hardcoded mapper location, parse from kpartx
+    devloop=$(losetup --list --noheadings | grep "$IMAGE_HOME/$IMAGE" | awk '{print $1}')
```

Comment:
:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

Reply:
Landed in master as #8097